pkgs/tools/admin/tigervnc/default.nix

```diff
@@ -76,31 +92,70 @@ stdenv.mkDerivation rec {
         --with-xkb-output=$out/share/X11/xkb/compiled
     make TIGERVNC_SRC=$src TIGERVNC_BUILDDIR=`pwd`/../.. -j$NIX_BUILD_CORES -l$NIX_BUILD_CORES
     popd
+  '' + lib.optionalString stdenv.isDarwin ''
+    make dmg
   '';
 
-  postInstall = ''
+  postInstall = lib.optionalString stdenv.isLinux ''
     pushd unix/xserver/hw/vnc
     make TIGERVNC_SRC=$src TIGERVNC_BUILDDIR=`pwd`/../../../.. install
     popd
     rm -f $out/lib/xorg/protocol.txt
 
     wrapProgram $out/bin/vncserver \
       --prefix PATH : ${lib.makeBinPath (with xorg; [ xterm twm xsetroot xauth ]) }
+  '' + lib.optionalString stdenv.isDarwin ''
+    undmg TigerVNC-$version.dmg
+    mkdir -p $out/Applications
+    mv 'TigerVNC Viewer ${version}.app' $out/Applications/
+    rm $out/bin/vncviewer
+    touch $out/bin/vncviewer
+    echo "#!/bin/bash" >> $out/bin/vncviewer
```

Comment:
```suggestion
    echo "#!/bin/bash" > $out/bin/vncviewer
```

Reply:
Fixed with c2bc312 in the latest force-push.